sdk/core/src/main/java/com/klaviyo/core/config/KlaviyoConfig.kt

```diff
@@ -161,13 +179,25 @@ object KlaviyoConfig : Config {
             KlaviyoConfig.applicationContext = applicationContext as Context
             KlaviyoConfig.debounceInterval = debounceInterval
             KlaviyoConfig.networkTimeout = networkTimeout
-            KlaviyoConfig.networkFlushInterval = networkFlushInterval
+            KlaviyoConfig.networkFlushIntervals = networkFlushIntervals
             KlaviyoConfig.networkFlushDepth = networkFlushDepth
             KlaviyoConfig.networkMaxRetries = networkMaxRetries
 
             return KlaviyoConfig
         }
     }
+
+    enum class NetworkTypes(val position: Int) {
+        WIFI(0),
+        CELL(1),
+        OFFLINE(2);
+
+        companion object {
+            fun fromPosition(position: Int) = NetworkTypes.values().first() {
```

Comment:
Would it be simpler to just have this as a raw enum and store the intervals as a `map<NetworkType, Int>`

Reply:
Can do. I considered both approaches. Just thought an array was simpler to implement but a map might be nicer overall.

I'm happy to make the swap if thats preferrable.

Reply:
TBH I'm happy either way, sometimes I'm just typing whatever comes into my head...